src/compiler/transformers/module/system.ts

```diff
@@ -120,6 +121,8 @@ namespace ts {
             );
 
             // Write the call to `System.register`
+            // Reset the source-file's nodeFlag to not emit helpers because we already set one in the body

```

Comment:
`Reset the source-file's nodeFlag` -> `Clear the emit-helpers flag for later passes since we'll have already used it in the module body`